sdk/python/jobs/automl-standalone-jobs/automl-forecasting-demand-forecasting-many-models/auto-ml-forecasting-demand-forecasting-many-models.ipynb

```diff
@@ -0,0 +1,1429 @@
+    "- Well-defined interface: Components require a well-defined interface (input and output). The interface allows the user to build steps and connect steps easily. The interface also hides the complex logic of a step and removes the burden of understanding how the step is implemented.\n",
+    "\n",
+    "- Share and reuse: As the building blocks of a pipeline, components can be easily shared and reused across pipelines, workspaces, and subscriptions. Components built by one team can be discovered and used by another team.\n",
+    "\n",
+    "- Version control: Components are versioned. The component producers can keep improving components and publish new versions. Consumers can use specific component versions in their pipelines. This gives them compatibility and reproducibility.\n",
+    "\n",
+    "For a more detailed information on this subject, refer to the this [link](https://learn.microsoft.com/en-us/azure/machine-learning/concept-component?view=azureml-api-2).\n",
+    "\n",
+    "To import components,  we need to get the registry. The following command obtains the public regsitry from which we will import components for our experiment."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false,
+    "gather": {
+     "logged": 1682992409837
+    },
+    "jupyter": {
+     "outputs_hidden": false,
+     "source_hidden": false
+    },
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# get registry\n",
+    "# TODO: Change registry name once the components are in the public registry\n",
+    "ml_client_registry = MLClient(\n",
+    "    credential=credential, registry_name=\"ManyModels_HTS_BugBash\"\n",
+    ")\n",
+    "print(ml_client_registry)\n",
+    "print(\"---\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we pull specific components and use them to build a pipeline of steps. For the illustration of the product evaluation workflow we will use the following components:\n",
+    "- Data partitioning component: allows users to partion the data for many models runs, both, training and inference.\n",
+    "- Many models training component: trains the best model per partition specified by users.\n",
```

Comment:
Maybe something like, "...train and select the best model for each partition separately (or, independently)" would be more friendly for users not familiar with many models already?